### PR TITLE
The test task

### PR DESCRIPTION
**Pull request by Aleksandern**
Hello.

Please review my result of the test task.

Regards,
Alex

### CODE REVIEW

**Comment by Aleksandern** [comment]
Video
https://youtu.be/sRX1EXDEp_c

**Comment by Aleksandern** [comment]
@azimgd

**Comment by Aleksandern** [comment]
@azimgd 
It's not relevant anymore?

File: src/normalizer/schemas.js [hunk 1]
```diff
@@ -276,6 +276,24 @@ const postSchema = new schema.Entity(
...
+	 */
+	'dating',
+	/**
+	 * Definition
+	 */
+	{
+	},
+	/**
+	 * Options
+	 */
```
Comment (mfogel) [inline src/normalizer/schemas.js:291]: I don't think these comments are necessary

File: src/normalizer/schemas.js [hunk 2]
```diff
@@ -371,3 +389,13 @@ export function denormalizeChatGet(payload, entities) {
 	return denormalize(payload, chatSchema, entities)
 }
 
+/**
+ *
+ */
```
Comment (mfogel) [inline src/normalizer/schemas.js:394]: Meaningless comment

File: src/normalizer/schemas.js [hunk 3]
```diff
@@ -43,7 +43,7 @@ const storyCommentsSchema = new schema.Entity(
 	/**
 	 * Key
 	 */
-	'comments', 
+	'comments',
```
Comment (mfogel) [inline src/normalizer/schemas.js:46]: I would try to avoid causing changes to lines unrelated to the PR

**Pull request closed without merging**